src/state/queries/notifications/types.ts

```diff
@@ -28,7 +28,7 @@ export interface FeedPage {
 }
 
 export interface CachedFeedPage {
-  sessDid: string // used to invalidate on session changes
+  usableInFeed: boolean // if true, the cached page is recent enough to use as the response
```

Comment:
Nit: could chuck this in a doc block and get it on hover where we use it

Reply:
Done